### DISCUSSION TITLE
換臉程式存在的必要性？

### DISCUSSION THREAD

**Discussion by charlottechuang** [1101-GS4719-C/Python-Community-GS4719-C#291] [科技新知]
近年很多換臉程式的誕生，甚至很多以「能夠把換臉照片做得毫無破綻」自豪(Deep fake技術逐漸成熟？），但我實在想不到除了惡搞的正面用途😅😅 去年甚至有人拿這個技術來把女明星的臉換到各種A片上盈利⋯。 
https://mrmad.com.tw/ai-face-swapper/amp

**Comment by 20010906** [comment]
說不定能讓對自己身材不自信的人，在身材好的照片換上自己的臉，多少安慰一點XD

**Comment by zixunn** [comment]
感覺不管是以前還是現在，不同技術的好與壞端看使用者如何使用。

**Comment by chun1203** [comment]
可能是要催眠自己變好看了（？）XD

**Comment by EricAgumonJou** [comment]
正面用途的話有常被好萊塢後製運用，像是電影《玩命關頭》男星保羅沃克因車禍身亡，劇組便邀請其弟弟代打，將保羅沃克的五官轉移到他弟弟臉上，加上聲音合成，讓保羅沃克在螢幕上「重生」另外在行銷市場上的用途。

**Comment by EricAgumonJou** [comment]
也有不少電商用Deepfake合成出「不存在這世界上的模特兒」讓這些假model接平面廣告，解決棚拍成本還能避免合約爭議。

**Comment by EricAgumonJou** [comment]
雖說現在有不少科技公司也推出相對應的演算法，查看影片中人物的皮膚顏色、眨眼次數等以揪出假影片或照片，但確實效果有限，期待未來該技術能更成熟～～

**Comment by cody910120** [comment]
就是會有人拿來做不肖的事情:)

**Comment by mesakiyuta** [comment]
感覺可以拿來跟整形行業結合，讓客人先看到最終成果，再決定是否要動刀

**Reply by charlottechuang** [reply]
這個我覺得滿實用的哈哈哈

**Comment by yt7l0** [comment]
剪頭髮或染頭髮之前也可以先看一下自己適不適合，以免花錢之後後悔

**Comment by Joan0518** [comment]
記得之前有一陣子出現韓劇主角可以直接換上你的臉，很好玩xdd，現在換臉的程式真的很多~~

**Comment by 109302013evantsao** [comment]
不論在哪個國家，總是會有人想試試看能否鑽看看漏洞，去鑽取利益，有時是真的走投無路只能犯罪，但多數時就只是心起歹念而已

**Comment by zhiron544** [comment]
換臉技術確實有很多創新和有趣的用途，但也有很多道德和法律的問題。我認為應該加強對這種技術的監管和規範，保護人們的隱私和權益

**Comment by 110302001** [comment]
我覺得不管哪個技術都會有拿去做壞事，但絕對不是因此就認為這技術不該存在，應該用法律或其他方式去限制和防範有人拿去不好的用途上。
